doc/source/management/extended_show_grants.rst

```diff
@@ -4,7 +4,7 @@
  Extended ``SHOW GRANTS``
 ==========================
 
-Prior to |Percona Server| :rn:`5.6.23-72.1`, ``SHOW GRANTS`` displays only the privileges granted explicitly to the named account. Other privileges might be available to the account, but they are not displayed. For example, if an anonymous account exists, the named account might be able to use its privileges, but ``SHOW GRANTS`` will not display them. In :rn:`5.6.23-72.1` ``SHOW GRANTS`` command was extended to display all the effectively available privileges to the account.
+Prior to |Percona Server| :rn:`5.7.10-1`, ``SHOW GRANTS`` displays only the privileges granted explicitly to the named account. Other privileges might be available to the account, but they are not displayed. For example, if an anonymous account exists, the named account might be able to use its privileges, but ``SHOW GRANTS`` will not display them. In :rn:`5.7.10-1` ``SHOW GRANTS`` command was extended to display all the effectively available privileges to the account.
```

Comment:
There is no "prior" here I' say. "In Oracle MySQL, ``SHOW GRANTS``..." And later "In Percona Server `SHOW GRANTS`` command was extended ... "